snmp-scan.py

```diff
@@ -120,7 +121,7 @@ def scan_host(ip):
             pass
 
         try:
-            add_output = check_output(['/usr/bin/env', 'php', 'addhost.php', hostname or ip])
```

Comment:
I feel link snmp-scan.py option should be allow ping only devices to be added, this way it tries snmp first.

Reply:
I think I prefer to keep it as ping only. If someone really want to do a combined scan they could start with the SNMP scan to add all SNMP devices and then do a ping scan to add the rest. Otherwise we would lose the ability to add devices as ping only if they happen to reply to SNMP.

It would also take longer time to scan it, but I don't think that matter much.